pkg/webui/console/store/actions/device.js

```diff
@@ -40,6 +41,10 @@ export const getDevice = (appId, deviceId, selector, options) => (
   { type: GET_DEV, appId, deviceId, selector, options }
 )
 
+export const patchDevice = (appId, deviceId, patch) => (
```

Comment:
why do we need this?

Reply:
To update (patch) the device entry in the store with the new data returned from the update api call.

Reply:
how is it different from updating application/gateway?

Reply:
we have `update` both in the js sdk and `api` module for the console. lets be consistent

Reply:
I chose `patch` here because it is only partially updating (= patching) the entity store, as opposed to updating it completely. It's right though that we use update in the SDK with patches as well. We should make sure then that it is clear that both patches as well as complete entities are allowed with this action.